cmake/aws-c-cal-config.cmake

```diff
@@ -2,14 +2,21 @@ include(CMakeFindDependencyMacro)
 
 find_dependency(aws-c-common)
 
-if (NOT BYO_CRYPTO AND NOT WIN32 AND NOT APPLE)
-    list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/modules")
-    find_dependency(crypto)
-endif()
-
 if (BUILD_SHARED_LIBS)
     include(${CMAKE_CURRENT_LIST_DIR}/shared/@PROJECT_NAME@-targets.cmake)
 else()
     include(${CMAKE_CURRENT_LIST_DIR}/static/@PROJECT_NAME@-targets.cmake)
 endif()
 
+if (NOT BYO_CRYPTO AND NOT WIN32 AND NOT APPLE)
+    get_target_property(AWS_C_CAL_DEPS AWS::aws-c-cal INTERFACE_LINK_LIBRARIES)
```

Comment:
aws-c-cal target is already entirely resolved by the time this script runs?

Reply:
Yes, it is resolved few lines above.